tests/controller/insertcontent.js

```diff
@@ -62,6 +62,57 @@ describe( 'DataController', () => {
 			expect( getData( doc ) ).to.equal( 'xa[]x' );
 		} );
 
+		describe( 'saves the reference to the original object', () => {
+			it( 'inline-widget', () => {
```

Comment:
It would be nice if the test name would create a sentence:

```
describe( 'insertContent' ...
it( 'should saves the reference to the original object' ...
```

It's not needed in all cases, for instance there is no reason to create strange sentences, if you test a function with a set of data, but here it's pretty nice and simple.

Also, I don't think we need a separate describe block here.